code/modules/mob/living/living_movement.dm

```diff
@@ -8,12 +8,12 @@
 	if(.)
 		return
 	if(mover.throwing)
-		return (!density || body_position == LYING_DOWN || (mover.throwing.thrower == src && !ismob(mover)))
+		return (!density || (body_position == LYING_DOWN) || (mover.throwing.thrower == src && !ismob(mover)))
 	if(buckled == mover)
 		return TRUE
 	if(ismob(mover) && (mover in buckled_mobs))
 		return TRUE
-	return !mover.density || body_position == LYING_DOWN
+	return !mover.density || (body_position == LYING_DOWN)
```

Comment:
? Is operator precedence this bad?

Reply:
@Jared-Fogle no not really i admittedly just have a habit of doing it because i can't memorize operator precedence that well